Link from work order to property.

pudiva opened the pull request:
Trello:
https://trello.com/c/Hp4FFE2g/206-as-an-rcc-agent-i-need-a-way-to-get-back-to-the-property-page-from-the-works-order-page

pudiva committed:
Link from work order to property.

cdccollins commented on app/views/work_orders/_work_order_reference_top_section.html.erb:
```diff
@@ -23,9 +23,11 @@
-          <% end %>
+          <span class="govuk-!-font-weight-bold">
+            <%= link_to property_path(work_order.property.reference), class: "govuk-link hackney-link" do %>
+              <%= sanitize format_address(work_order.property.address).join("<br>") %>
```
<img width="470" alt="Screenshot 2019-06-19 at 09 42 36" src="https://user-images.githubusercontent.com/35098639/59750365-9d0a3980-9276-11e9-9842-59c282cd54f6.png">
I got an extra space after the link address when I tried this out

pudiva replied:
Strange, I got this too and fixed it. Gonna take a look!

pudiva replied:
@cdccollins looks like we found a bug in chromium. Firefox doesn't render this space at the end. I've fixed by `strip`ing and passing as string parameter instead of block to `link_to`.

cdccollins replied:
<img width="406" alt="Screenshot 2019-06-19 at 11 54 04" src="https://user-images.githubusercontent.com/35098639/59759896-fa0eeb00-9288-11e9-8fca-73cc375045dc.png">

pudiva replied:
```
<html>
  <body>
    a<a href="https://www.youtube.com/watch?v=3sXU2YqUKOY"> a </a>a
  </body>
</html>
```

@cdccollins this will render underlined spaces at the beginning and at the end of the link. I think this is a bug on chromium. Wanna report it?

pudiva replied:
> <img alt="Screenshot 2019-06-19 at 11 54 04" width="406" src="https://user-images.githubusercontent.com/35098639/59759896-fa0eeb00-9288-11e9-8fca-73cc375045dc.png">

I couldn't reproduce this one.

cdccollins replied:
That's what I now get when I test it

pudiva replied:
Actually the above html renders exactly the same on both Firefox and Chromium. It needs a `<br>` to render differently, like this:

```
<html>
  <body>
    a<a href="https://www.youtube.com/watch?v=3sXU2YqUKOY"> a </a><br>a
  </body>
</html>
```

pudiva committed:
Fix trailing space on property link in work_order.

For some reason chromium renders an underlined space at the end of the
link. Firefox is not affected.
